packages/components/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,221 @@
+                                  <Menu.Item
+                                      key={option.title}
+                                      title={option.title}
+                                      active={option.active}
+                                      closeMenuOnClick={false}
+                                      iconName={option.iconName}
+                                      onPress={() => {
+                                          handleToggleOptionPress(option.title);
+                                          option.onPress();
+                                      }}
+                                  />
+                              ))
+                            : filteredOptions.map((option, index) => (
+                                  <Menu.Item
+                                      key={index}
+                                      title={option}
+                                      onPress={() => {
+                                          setInputValue(option);
+                                          onSelect(option);
+                                          setIsOptionsVisible(false);
+                                      }}
+                                  />
+                              ))}
+                    </ScrollView>
+
+                    {/* <FlatList
+                        data={filteredOptions}
+                        keyExtractor={item => item}
+                        renderItem={({ item }) => (
+                            <TouchableOpacity
+                                style={styles.option}
+                                onPress={() => {
+                                    setInputValue(item);
+                                    onSelect(item);
+                                    setIsOptionsVisible(false);
+                                }}
+                            >
+                                <Text style={styles.optionText}>{item}</Text>
+                            </TouchableOpacity>
+                        )}
+                    /> */}
+                </Menu>
+            )}
+        </View>
+    );
+};
+
+const themedStyles = EDSStyleSheet.create(
+    (theme, { inputLayout }: { inputLayout?: LayoutRectangle }) => ({
+        container: {
+            // styles for the container of the component
+        },
+        option: {
+            padding: 15,
+            borderColor: theme.colors.text.tertiary,
+        },
+        optionText: {
+            color: theme.colors.text.primary,
+            ...theme.typography.basic.input,
+            fontSize: 18,
+        },
+        menuContainer: {
+            maxHeight: 300,
+            width: inputLayout?.width || "100%",
+        },
+        adornmentContainer: {
+            flexDirection: "row",
+            alignItems: "center",
+        },
+        closeIcon: {
+            marginLeft: 8,
+        },
+        menuIcon: {
+            marginRight: 3,
+        },
+    }),
+);
```

Comment:
it looks like `option` and `optionText` is not being used. Those styles can be removed